README.md

```diff
@@ -399,16 +373,24 @@ Note: Do not need to worry about Quality or Format for this application.
 
 The Digital Collections app runs unit tests with Jest and React Testing Library.
 
+We have two groups of tests: pages/components have unit tests in the `jsdom` environment, and the API endpoints are tested in the `node` environment.
```

Comment:
Can you add that the pages/components and `jsdom` tests are for the react/client side tests and the API/`node` env tests are for the functional ones and mostly due to how the app is structured in Next (because we _have_ to do this in the new Next.js App router architecture, right?)